### PR TITLE
AWS API Gateway: Clarify access

### PR DESCRIPTION
**Pull request by chids**
Point out that resources can't be made "network internal" but that they may requrie authentication.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1069,6 +1069,7 @@ API Gateway
 
 -	🔸API Gateway only supports encrypted (https) endpoints, and does not support unencrypted HTTP. (This is probably a good thing.)
 -	🔸API Gateway endpoints are public — there is no mechanism to build private endpoints, e.g. for internal use.
```
Comment (ThanosBaskous) [inline README.md]: I think that this point has been made redundant by your addition - can you remove it?

Comment (chids) [inline README.md]: 😄 right, a git merge mishap....fixed in eecf85b

**Comment by ThanosBaskous** [comment]
LGTM, @chids - thank you! Bouncing over to @jlevy for second stage review.

**Comment by jlevy** [comment]
👍 LGTM too. Thanks!

**Pull request merged**